
A GCC cross compiler does not yield a libgcc_eh.so. The GCC and GlibC team
should smoke their good stuff together (and the GlibC freaks be fet with a
book about release management ...).

  - Rene Rebe <rene@exactcode.de>

--- libc/Makeconfig.orig	2004-12-31 20:33:44.918457704 +0100
+++ libc/Makeconfig	2004-12-31 20:55:54.452337832 +0100
@@ -514,12 +514,12 @@
   libunwind = -lunwind
 endif
 ifneq ($(have-as-needed),yes)
- libgcc_eh := -lgcc_eh $(libunwind)
+ libgcc_eh := $(libunwind)
 else
  libgcc_eh := -Wl,--as-needed -lgcc_s$(libgcc_s_suffix) $(libunwind) -Wl,--no-as-needed
 endif
 gnulib := -lgcc $(libgcc_eh)
-static-gnulib := -lgcc -lgcc_eh $(libunwind)
+static-gnulib := -lgcc $(libunwind)
 libc.so-gnulib := -lgcc
 endif
 ifeq ($(elf),yes)
